package.json

```
@@ -42,7 +44,7 @@
     }
   },
   "scripts": {
-    "build": "npm run uglify && npm run makepot && npm run sass",
+    "build": "npm install && composer install && npm run uglify && npm run makepot && npm run sass && node tasks/release.js && mv release/woocommerce-gateway-stripe.zip .",
```

What do you think of splitting this to improve readability? Something along the lines of:

```suggestion
    "prebuild": "npm install && composer install",
    "build": "npm run uglify && npm run makepot && npm run sass && npm run build:release",
    "build:release": "node tasks/release.js && mv release/woocommerce-gateway-stripe.zip .",
```